nvtabular/loader/torch.py

```diff
@@ -174,8 +174,38 @@ def _get_sparse_tensor(self, values, indices, num_rows, seq_limit):
             sparse_tensor = sparse_tensor.to_dense()
         return sparse_tensor
 
-    def _build_sparse_tensor(self, values, offsets, diff_offsets, num_rows, seq_limit):
+    def _build_sparse_tensor(
+        self, values, offsets, diff_offsets, num_rows, seq_limit, padding: str = ""
+    ):
+        """Builds sparse tensors in our torch dataloader.
+
+        Parameters
+        ----------
+        values :
+        offsets :
+        diff_offsets :
+        num_rows :
+        seq_limit :
+        padding : str, optional
+            Padding mode, choose among 'left' for left padding, 'right' for right padding,
+            or '' for no padding, by default ''
+
+        Returns
+        -------
+        torch.sparse
+            Our built torch sparse tensor.
+
+        Raises
+        ------
+        NotImplementedError
+            Raises this error when this method is called with a not implemented
+           padding mode string.
+        """
         indices = self._get_indices(offsets, diff_offsets)
+        if padding == "left":
+            indices[:, 1] = seq_limit - 1 - indices[:, 1]
+        if padding == "right":
+            raise NotImplementedError
```

Comment:
From the original issue from Gabriel, it sounded to me, based on what was written there, that right padding has already been implemented. I did not want to reduplicate it here to avoid doing the same thing in multiple places of the codebase. Let me investigate some more whether this has been already implemented, and hence should not be duplicated, or whether it makes sense to implement this feature here.

Reply:
@benfred Would you know off-hand, based on your knowledge of the codebase, whether right padding has definitely been or not been implemented elsewhere in the repository?